ansible/roles/collectory/templates/config/collectory-config.properties

```diff
@@ -16,7 +16,7 @@ security.cas.uriFilterPattern={{ collectory_uri_filter_pattern | default('/admin
 security.cas.uriFilterPattern={{ collectory_uri_filter_pattern | default('/admin,/admin/*,/collection/*,/institution/*,/contact/*,/reports/*,/providerCode/*,/providerMap/*,/dataProvider/*,/dataResource/*,/dataHub/*,/manage/*,/alaAdmin,/alaAdmin/*,/ws/ipt/scan/*') }}
 {% endif %}
 
-security.cas.authenticateOnlyIfLoggedInFilterPattern={{ collectory_authenticate_only_if_logged_in_filter_pattern | default('/public/show/*') }}
+security.cas.authenticateOnlyIfLoggedInFilterPattern={{ collectory_authenticate_only_if_logged_in_filter_pattern | default('') }}
```

Comment:
The removal of /public/show/* breaks the functionality of this page. There is role conditional content and without this pattern it breaks the login button that would otherwise be required to make it work.